media_driver/cmake/linux/media_compile_flags_linux.cmake

```diff
@@ -38,29 +38,13 @@ set(MEDIA_COMPILER_FLAGS_COMMON
     -Werror=return-type
 
     # General optimization options
-    -march=${UFO_MARCH}
     -mpopcnt
```

Comment:
popcnt is only available with SSE4.2, so please also drop that flag.

Reply:
yep. thank's for the catch. @egorovdanil : same idea - try to drop and check whether we use these instructions anywhere in the code.

@sebastinas : mind that this PR is only the first step. We likely need similar change in gmmlib.

Reply:
done